es/03_4_Recibiendo_una_Transaccion.md

```diff
@@ -0,0 +1,333 @@
+    "label": "",
+    "vout": 0,
+    "confirmations": 1,
+    "blockhash": "00000000000001753b24411d0e4726212f6a53aeda481ceff058ffb49e1cd969",
+    "blockheight": 1772396,
+    "blockindex": 72,
+    "blocktime": 1592600085,
+    "txid": "ca4898d8f950df03d6bfaa00578bd0305d041d24788b630d0c4a32debcac9f36",
+    "walletconflicts": [
+    ],
+    "time": 1592599938,
+    "timereceived": 1592599938,
+    "bip125-replaceable": "no"
+  }
+]
+
+```
+
+Aquí se muestran dos transacciones (`8e2ab10cabe9ec04ed438086a80b1ac72558cc05bb206e48fc9a18b01b9282e9`) y (`ca4898d8f950df03d6bfaa00578bd0305d041d24788b630d0c4a32debcac9f36`) para una cantidad específica ( `0.01000000` y` 0.00010000`), que tanto se recibieron ( `receive`) por la misma dirección en nuestra cartera (`mi25UrzHnvn3bpEfFCNqJhPWJn5b77a5NE`) . Eso es una mala higiene de claves, por cierto: debe usar una nueva dirección para cada Bitcoin que reciba. En este caso, nos impacientamos porque el primer grifo no parecía funcionar.
+
+Puede acceder a información similar con el comando `bitcoin-cli listunspent`, pero solo muestra las transacciones por el dinero que no ha gastado. Estos se denominan UTXO y serán de vital importancia cuando envíe dinero al mundo de Bitcoin:
+
+```
+$ bitcoin-cli listunspent
+[
+  {
+    "txid": "ca4898d8f950df03d6bfaa00578bd0305d041d24788b630d0c4a32debcac9f36",
+    "vout": 0,
+    "address": "mi25UrzHnvn3bpEfFCNqJhPWJn5b77a5NE",
+    "label": "",
+    "scriptPubKey": "76a9141b72503639a13f190bf79acf6d76255d772360b788ac",
+    "amount": 0.00010000,
+    "confirmations": 1,
+    "spendable": true,
+    "solvable": true,
+    "desc": "pkh([d6043800/0'/0'/1']02fd5740996d853ea51a6904cf03257fc11204b0179f344c49739ec5b20b39c9ba)#62rud39c",
+    "safe": true
+  },
+  {
+    "txid": "8e2ab10cabe9ec04ed438086a80b1ac72558cc05bb206e48fc9a18b01b9282e9",
+    "vout": 1,
+    "address": "mi25UrzHnvn3bpEfFCNqJhPWJn5b77a5NE",
+    "label": "",
+    "scriptPubKey": "76a9141b72503639a13f190bf79acf6d76255d772360b788ac",
+    "amount": 0.01000000,
+    "confirmations": 1,
+    "spendable": true,
+    "solvable": true,
+    "desc": "pkh([d6043800/0'/0'/1']02fd5740996d853ea51a6904cf03257fc11204b0179f344c49739ec5b20b39c9ba)#62rud39c",
+    "safe": true
+  }
+]
+```
+
+Tenga en cuenta que los bitcoins no son solo un lío homogéneo de dinero en efectivo atascado en su bolsillo. Cada transacción individual que recibe o envía se coloca en el libro mayor inmutable de blockchain, en un bloque. Puede ver estas transacciones individuales cuando mira su dinero no gastado. Esto significa que el gasto en bitcoins no es tan anónimo como parece. Aunque las direcciones son bastante privadas, las transacciones se pueden examinar a medida que entran y salen de las direcciones. Esto hace que la privacidad sea vulnerable al análisis estadístico. También introduce cierta no fungibilidad potencial para bitcoins, ya que puede rastrear una serie de transacciones, incluso si no puede rastrear un "bitcoin" específico.
```

Comment:
- `dinero en efectivo apelmazado/apretado en su bolsillo`
- `También introduce cierta no fungibilidad potencial para bitcoins, ya que puede rastrear una...` ---> `También introduce cierta no fungibilidad potencial para bitcoins, ya que usted puede rastrear una...`

Reply:
atascado is better for me.
Usted is implicit.